flask_stormpath/__init__.py

```diff
@@ -274,9 +319,13 @@ def application(self):
         ctx = stack.top
         if ctx is not None:
             if not hasattr(ctx, 'stormpath_application'):
-                ctx.stormpath_application = self.client.applications.search(
```

Comment:
We should prolly leave this original client behavior alone. This is going to need to be refactored in a different way in the near future.